kraken_sdk_futures_rest/src/client.rs

```diff
@@ -81,10 +92,11 @@ impl Default for Client {
 }
 
 impl Client {
-    pub fn new(public_key: &str, private_key: &str) -> Self {
+    pub fn new(public_key: &str, private_key: &str, timeout: Option<Duration>) -> Self {
```

Comment:
We should not change the interface of the `new` constructor. The user can set a custom timeout using the builder interface. Can you please change this? (here and on the other client).